components/speedreader/rust/ffi/src/speedreader.cc

```diff
@@ -0,0 +1,149 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "../include/speedreader.h"
+
+#include <iostream>
+#include "base/logging.h"
+
+#include "../include/speedreader_ffi.h"
+
+#define UNUSED (void)
```

Comment:
what is this for?

Reply:
leftover from testing when simple test callbacks were not using all arguments. Removing now